js-api-doc/compile.d.ts

```diff
@@ -99,12 +187,14 @@ export function compileAsync(
  *
  * This only allows synchronous {@link Importer}s and {@link CustomFunction}s.
  *
- * **Heads up!** When using the `sass-embedded` npm package,
+ * **Heads up!** When using the [sass-embedded] npm package,
```

Comment:
@nex3 We may want to see if this is still true once the implementation is complete. The [initial benchmarking](https://github.com/sass/embedded-host-node/pull/261#issuecomment-1811187248) from @jerivas showed the difference between sync and async may now be negligible.

Reply:
We'll need to test something that's more chatty... it makes sense that the difference would be very small when most of each compilation time is being spent in the compiler, but once a bunch of messages are being sent for each compilation that may change the equation.

Reply:
We are now testing 1000 concurrent compilations on https://github.com/sass/sass-spec/pull/1954

Reply:
What's the result performance-wise?

Reply:
Async is much slower in dart version mainly because heavy use of `FutureOr`. If you test with extremely simple and small input it won't show much difference but if you try to benchmark something large, e.g. bootstrap, you will still find async to be way slower.

Reply:
I'm specifically asking about the embedded host's performance in async versus sync mode. Either way, Dart Sass is running in sync mode.

Reply:
I ran a new set of benchmarks here https://github.com/sass/embedded-host-node/pull/261#issuecomment-1857084145